### PR TITLE
Backend consistency in handling of empty tensor in `min` and `max`

### PR DESCRIPTION
**Pull request by guillaumebaquiast**
Correct a slight inconsistency between Torch and the other backends in the handling of empty tensors for `max` and `min`:

- All backends but Torch follow the numpy behaviour, which is to throw an exception on empty tensor, except if the argument `initial` is passed. See screenshot 1, on JAX as an example.
- On backend Torch, it returns 0 for `max`, and it fails for `min`, irrespective of the argument `initial`. See screenshot 2 for `max`.

This PR aligns the behaviour of Torch backend on numpy (see screenshot 3). Also adds a test.

Screenshot 1 - JAX `max`
Pasted Graphic

Screenshot 2 - Torch `max`, before PR
Pasted Graphic 1

Screenshot 3 - Torch `max`, after PR
Pasted Graphic 2

### CODE REVIEW

File: keras_core/backend/torch/numpy.py [hunk 1]
```diff
@@ -550,6 +556,14 @@ def meshgrid(*x, indexing="xy"):
 
 def min(x, axis=None, keepdims=False, initial=None):
     x = convert_to_tensor(x)
+    if 0 in x.shape:
+        if initial is None:
+            raise ValueError("Cannot compute the max of an empty tensor.")
```
Comment (fchollet) [inline keras_core/backend/torch/numpy.py:561]: Typo: min

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: keras_core/backend/torch/numpy.py (hunk 1 above)
Comment (guillaumebaquiast) [reply keras_core/backend/torch/numpy.py]: thanks

**Review by fchollet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**